src/hooks/useCourtContracts.js

```diff
@@ -174,11 +197,110 @@ export function useDisputeActions() {
     [disputeManagerContract]
   )
 
-  return { draft, commit, reveal, leak, appeal, confirmAppeal }
+  const executeRuling = useCallback(
+    disputeId => {
+      return aragonCourtContract.executeRuling(disputeId, {
+        gasLimit: GAS_LIMIT,
+      })
+    },
+    [aragonCourtContract]
+  )
+  return {
+    approveFeeDeposit,
+    draft,
+    commit,
+    reveal,
+    leak,
+    appeal,
+    confirmAppeal,
+    executeRuling,
+  }
+}
+
+/**
+ *
+ * @param {string} disputeId id of the dispute
+ * @param {string} roundId id of the round
+ * @returns {Object} appeal deposit and confirm appeal deposit amounts
+ */
+export function useAppealDeposits(disputeId, roundId) {
+  const [appealDeposits, setAppealDeposits] = useState([bigNum(0), bigNum(0)])
+
+  const disputeManagerContract = useCourtContract(
+    CourtModuleType.DisputeManager,
+    disputeManagerAbi
+  )
+
+  useEffect(() => {
+    const getNextRoundDetails = async () => {
```

Comment:
Should we call this `nextRoundDetails()` since it doesn’t return anything?

Reply:
What about `fetchNextRoundDetails` for consistency with other functions that have the same functionality e.g `fetchTotalActiveBalance`